packages/components/src/draggable/stories/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Draggable from '../';
+import Dashicon from '../../dashicon';
+
+export default { title: 'Draggable', component: Draggable };
+
+const Box = ( props ) => {
+	return (
+		<div
+			{ ...props }
+			style={ {
+				alignItems: 'center',
+				display: 'flex',
+				justifyContent: 'center',
+				width: 100,
+				height: 100,
+				background: '#ddd',
+			} }
+		/>
+	);
+};
+
+export const _default = () => {
+	const [ isDragging, setDragging ] = useState( false );
```

Comment:
According to documentation, the function exported as a story isn't a React component. It means that we shouldn't use hooks in this context as they might stop to work in the near future if they change the way all that works.

The easiest way to get rid of this warning is to move the part which manages state out of this function. See a similar usage:
https://github.com/WordPress/gutenberg/blob/a6e1252886e8d990627ac4bf4796f768bfad2a44/packages/components/src/checkbox-control/stories/index.js#L18-L28

In this case, it might be more work to do it :(